server/auth/authorization.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const app = express();
+const jwt = require('jsonwebtoken');
+const dotenv = require('dotenv');
+
+
+let refreshTokens = [];
+app.use(express.json());
+dotenv.config();
+
+function auth(req, res, next){
+    let token = req.headers['authorization'];
+    token = token.split(' ')[1];
+
+    jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
+        if(!err){
+            req.user = user;
+            next();
+        } else {
+            return res.status(403).json({ message: 'User not authenticated' });
+        }
+    });
+}
+
+app.post('/renewAccessToken', (req, res) => {
```

Comment:
Please make endpoints with dashes or in a path style. 
And read specifications on how to make better endpoint naming.
https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/